test/libsolidity/syntaxTests/functionCalls/internal_functions_with_calldata_args_assigned_to_function_pointers_of_same_type.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+    function g(bytes calldata b) pure internal returns (bytes calldata) {
+        return b[2:5];
+    }
```

Comment:
I think that `syntaxTests/functionTypes/` would be a better location for these tests. It's more about pointers than about calls.

Reply:
Done.